Update pickup-locations.md (#8308)
magento/devdocs: src/guides/v2.4/graphql/queries/pickup-locations.md

```

**Response:**

> For the available Pick Up location, The Response would be looks like.

Reviewer: ```suggestion
For the available Pickup location, the response would look like:
```

  }
}
```
If the Pickup location is not available for the given search term, the response will be empty:

```
{
  "data": {
    "pickupLocations": {
      "items": [],
      "total_count": 0,
      "page_info": {
        "page_size": 1,
        "current_page": 1,
        "total_pages": 0
      }
    }
  }
}

> ```

Reviewer 2: We don't need to show an empty response. Remove all of your additions up to here.

}
```

> If Pickup location will not available for the given search term, The Response will be emtpy,

Reviewer: ```suggestion
If the Pickup location is not available for the given search term, the response will be empty:
```

Attribute | Data type | Description
--- | --- | ---
`radius` | Int! | The radius to the search, in kilometers.

> `search_term` | String! | A combination of either the region, city, or postcode, a colon, and the country code. This value determines the location to use as the center of the search radius.  Valid search term - `Texas:US`|`Austin:US`|`78740:US`. After the colon, the two-letter Country code is required to be passed in uppercase.

Reviewer 2: I'm not sure what you're trying to say here. Those | symbols can't be in a markdown table. 

Perhaps reword to 

Valid search terms include `Texas:US`, `Austin:US`, and `78740:US`. The two-letter country code must be uppercase

Attribute | Data type | Description
--- | --- | ---
`radius` | Int! | The radius to the search, in kilometers.

> `search_term` | String! | A combination of either the region, city, or postcode, a colon, and the country code. This value determines the location to use as the center of the search radius.  Valid Search Term, `Texas:US`|`Austin:US`|`78740:US`. After Colon, You must required to pass Two-letter Country code in Upper case.

Reviewer: ```suggestion
`search_term` | String! | A combination of either the region, city, or postcode, a colon, and the country code. This value determines the location to use as the center of the search radius.  Valid search term - `Texas:US`|`Austin:US`|`78740:US`. After the colon, the two-letter Country code is required to be passed in uppercase.
```
